RFCs/0070-Fn::Size.md

```diff
@@ -0,0 +1,99 @@
+# RFC for Fn::Size
+
+* **Original Author(s):**: @juegong2
+* **Tracking Issue**: https://github.com/aws-cloudformation/cfn-language-discussion/issues/70
+
+# Summary
+
+We will support an intrinsic function called Fn::Size that will return the number of elements in a given list.
+
+# Motivation
+
+CloudFormation users may want to get the number of elements in a given list to set certain resource parameters or generate conditions. It may also help to check if index is out of bound for !Select function. This RFC proposes to support the use case of getting the number of elements of a given list.
+
+* Reference:
+  * https://github.com/aws-cloudformation/cfn-language-discussion/issues/61
+  * https://github.com/aws-cloudformation/cfn-language-discussion/issues/37
```

Comment:
Reading through this issue, I'm unclear how `Fn::Size` solves the customer problem described. From the comments, it seems like there was consensus that `Fn::For` would be a cleaner solution to the original problem. Do you have an example of how `Fn::Size` would be used to solve the problem described in this specific issue?

Reply:
No I don't. But Fn::Size is suggested as a solution in this issue. Therefore I put the issue here.